html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js

```diff
@@ -393,6 +393,15 @@ const actions = {
     }).catch(err => {
       commit('NODE_ERROR', err.response)
     })
+  },
+  bulkDelete: ({ commit }, data) => {
+    commit('NODE_REQUEST')
+    return api.bulkDelete(data).then(response => {
+      commit('NODE_BULK_SUCCESS', response)
```

Comment:
We need a new mutation (`NODE_BULK_DELETED`), which does the same thing as `NODE_DESTROYED` and `NODE_NOT_EXISTS` but for the list of deleted nodes.